bindings/python/pinocchio/visualize/panda3d_visualizer.py

```diff
@@ -0,0 +1,129 @@
+import multiprocessing
+import numpy as np
+import os
+import sys
+import uuid
+import warnings
```

Comment:
```suggestion
import numpy as np
import os
import sys
import warnings
```

Reply:
`uuid` may also be removed (depending on the  other comment).

Reply:
ok, fixed

Reply:
fixed